fast64_internal/utility.py

```diff
@@ -754,6 +755,23 @@ def get_obj_temp_mesh(obj):
         if o.get("temp_export") and o.get("instanced_mesh_name") == obj.get("instanced_mesh_name"):
             return o
 
+def apply_objects_transformations(allObjs: Iterable[bpy.types.Object]):
```

Comment:
```suggestion
def apply_objects_modifiers_and_transformations(allObjs: Iterable[bpy.types.Object]):
```
ok this isn't the best name but at least it's 100% descriptive rather than 50% pepega

Reply:
@thecozies madge you don't like my name?
if no you can merge this

Reply:
oops! i like the name, just didnt see your comment, gonna post another update to this PR